### PR TITLE
Migrate live-ajax-search example to v4

### PR DESCRIPTION
**Pull request by raxell**
The API now requires a key, I've generated one and hard-coded it, let me know if it's ok.  
Live example of the changes [here](https://next.plnkr.co/edit/6fKVDP5TKJYtJ7ll)

### CODE REVIEW

**Comment by GianlucaGuarini** [comment]
Wow well done thank you

File: live-ajax-search/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          clearTimeout(timeout);
+          timeout = setTimeout(() => fn(...args), wait);
+        };
+      }
+
+      (async () => {
+        await riot.compile()
+
+        riot.mount('search')
+      })
```
Comment (raxell) [inline live-ajax-search/index.html:32]: Oops, I forgot to execute the function, can you fix it without making a new PR? @GianlucaGuarini

Comment (GianlucaGuarini) [reply live-ajax-search/index.html:32]: if you want I can grant you access to this repo

Comment (raxell) [reply live-ajax-search/index.html:32]: Would be great, thanks

**Pull request merged**